build/SignConfig.xml

```diff
@@ -3,12 +3,12 @@
   <job platform="" configuration="" dest="__INPATHROOT__" jobname="dep.controls" approvers="">
     <!-- Overwrite the input files to simplify downstream things like nuget and framework package generation. -->
     <file src="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
```

Comment:
>WindowsRT [](start = 107, length = 9)

is this the right signtype? the request seems failed.

Reply:
I've been using these lines as the indicator of successful / failure of the signing:

2020-10-05T04:34:23.8219582Z H:Completed . Saved as extracted files.
2020-10-05T04:34:23.8228673Z H:Moving from C:\Users\pkgescsadmin\AppData\Local\Temp\tmp4233\PkgESSignFilesa8db79d4-d29a-4a4f-8fca-767e664d66a8\Microsoft.ProjectReunion.dll to C:\Users\pkgescsadmin\AppData\Local\Temp\tmp4232\PkgESSignFilesa8db79d4-d29a-4a4f-8fca-767e664d66a8\Microsoft.ProjectReunion.dll
2020-10-05T04:34:23.8236579Z H:Hash of C:\Users\pkgescsadmin\AppData\Local\Temp\tmp4232\PkgESSignFilesa8db79d4-d29a-4a4f-8fca-767e664d66a8\Microsoft.ProjectReunion.dll EXPECTED: a1bf6e7cbc74bf1a2030d7756732a083d117b12c0b5753a711b1f47fe233a335, ACTUAL: a1bf6e7cbc74bf1a2030d7756732a083d117b12c0b5753a711b1f47fe233a335